docs/user-guide/logging/overview.md

```diff
@@ -0,0 +1,107 @@
+
+When you run [`kubectl logs`](/docs/user-guide/kubectl/kubectl_logs), as in the basic logging example, the kubelet on the node handles the request and reads directly from the log file, returning the contents in the response. Note that `kubectl logs` **only returns the last rotation**; you must manually extract prior rotations, if desired.
+
+### System components logs
+
+Kubernetes system components use a different logging mechanism than the application containers in pods. Components such as `kube-proxy` (among others) use the [glog](https://godoc.org/github.com/golang/glog) logging library. You can find the conventions for logging severity for those components in the [development docs on logging](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/logging.md).
+
+System components write directly to log files in the `/var/log` directory in the node's host filesystem. Like container logs, system component logs are rotated daily and based on size. However, system component logs have a higher size retention: by default, they store 100MB.
+
+## Cluster-level logging architectures
+
+While Kubernetes does not provide a native solution for cluster-level logging, there are several common approaches you can consider:
+
+* You can use a node-level logging agent that runs on every node.
+* You can include a dedicated sidecar container for logging in an application pod.
+* You can push logs directly to a back-end, from within an application.
+
+### Using a node logging agent
+
+![Using a node level logging agent](/images/docs/user-guide/logging/logging-with-node-agent.png)
+
+You can implement cluster-level logging by including a _node-level logging agent_ on each node. The logging agent is a dedicated tool that exposes logs or pushes logs to a back-end. Commonly the logging agent is a container that has access to a directory with log files from all of the application containers on that node.
+
+Because the logging agent must run on every node, it's common to implement it as either a DaemonSet replica, a manifest pod or a dedicated native process on the node. However the latter two approaches are deprecated and highly discouraged.
+
+Using a node-level logging agent is the most common and encouraged approach for a Kubernetes cluster, since it creates only one agent per node and it doesn't require any changes to the applications running on the node. However, node-level logging _only works for applications' standard output and standard error_.
+
+Kubernetes doesn't specify a logging agent, but there are two options, packaged with the kubernetes release: [Stackdriver Logging](/docs/user-guide/logging/stackdriver) for use with Google Cloud Platform, and [Elasticsearch](/docs/user-guide/logging/elasticsearch). You can find more information and instructions in the dedicated documents. Both use [fluentd](http://www.fluentd.org/) with custom configuration as an agent on the node.
```

Comment:
"Kubernetes doesn't specify a logging agent, but two optional logging agents are packaged with the Kubernetes release:"